test/org/sagebionetworks/bridge/play/controllers/AuthenticationControllerMockTest.java

```diff
@@ -208,11 +207,10 @@ public void signUpWithCompleteUserData() throws Exception {
         
         verify(authenticationService).signUp(eq(study), participantCaptor.capture());
         
-        StudyParticipant originalParticipant = TestConstants.PARTICIPANT;
+        StudyParticipant originalParticipant = TestUtils.getStudyParticipant(AuthenticationControllerMockTest.class);
         StudyParticipant persistedParticipant = participantCaptor.getValue();
         assertEquals(originalParticipant.getFirstName(), persistedParticipant.getFirstName());
         assertEquals(originalParticipant.getLastName(), persistedParticipant.getLastName());
-        assertEquals(originalParticipant.getEmail(), persistedParticipant.getEmail());
```

Comment:
Changed test so this assertion could be added back in (just need to not call getParticipant() twice as it creates a dummy participant with a random email.